MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionPresenter.h

```diff
@@ -182,7 +182,7 @@ protected slots:
                               Mantid::API::MatrixWorkspace_sptr &vanCurvesWS);
 
   // plots workspace according to the user selection
-  void plotFocusedWorkspace(std::string outWSName, std::string bank);
+  void plotFocusedWorkspace(std::string outWSName, int bank);
```

Comment:
Also, the name `bank` is now very misleading as it now is a plot sequence number or something like that.